CONTRIBUTING.md

```diff
@@ -89,12 +92,22 @@ We have automated systems in place for notifying our community translators that
 
 Our Docs are translated into several languages and we rely on automation to notify our translators that English pages have changed. By default, when a PR to an English page is merged, our Translation Status Tracker is updated.  
 
-**When choosing a PR title, please consider whether your PR should or should NOT trigger a rewrite to pages in other languages**: some tiny fixes are English only (e.g. spelling of English words) but some *will* require updating all language pages (e.g. a small error in a code sample).  Please use the word "minor" in  your PR title to override this default behavior and indicate that your PR does **NOT** require translating.
+**When choosing a PR title, please consider whether your PR should or should NOT trigger a rewrite to pages in other languages**: some tiny fixes are English only (e.g. spelling of English words) but some *will* require updating all language pages (e.g. a small error in a code sample).  Please use the word "minor" in your PR title to override this default behavior and indicate that your PR does **NOT** require translating.
```

Comment:
```suggestion
**When choosing a PR title, please consider whether your PR should or should NOT trigger a rewrite to pages in other languages**: some tiny fixes are English only (e.g. spelling of English words) but some *will* require updating all language pages (e.g. a small error in a code sample).  Please use the word "typo" in your PR title to override this default behavior and indicate that your PR does **NOT** require translating. Even if the change isn't *technically* a typo, this is a handy way for us all to think of PRs as relevant only to the English page, or to all, translated pages!
```
Sarah's going rogue! What do you think of this?  Otherwise, the whole PR LGTM!

Reply:
`minor` is also a valid keyword and avoids the "not technically a typo" problem, perhaps I didn't understand the point here. I'd say even to change this to `minor: PR title`, so we are very explicit that we are opting for removing the PR from updating the overview issue. What do you think?

```suggestion
**When choosing a PR title, please consider whether your PR should or should NOT trigger a rewrite to pages in other languages**: some tiny fixes are English only (e.g. spelling of English words) but some *will* require updating all language pages (e.g. a small error in a code sample). Please prefix your PR title with `minor: (PR title...)` to override this default behavior and indicate that your PR does **NOT** require translating. This is a handy way for us all to think of PRs as relevant only to the English page, or to all, translated pages!
```

Reply:
Because Sarah doesn't like that we use "minor" at all, actually, because people keep PRing "minor" (ie SMALL) changes.

I was hoping that using a word that really emphasized, "Oh, yah, this is something to fix the English site only" might be a better keyword to put in people's minds. If I had my magic wand, I'd remove using the word "minor" for this purpose entirely.  ;)

Reply:
And, I even LIKE that we explain, "Even if it's not technically a typo" because it gives us a chance to tell a better "story" and I think that "sticks" when people read it.

BUT, I am open to another word entirely that actually means ENGLISH ONLY NO TRANSLATE, and doesn't need explaining.

Reply:
> If I had my magic wand, I'd remove using the word "minor" for this purpose entirely. ;)

We can do that! It's a `minor` (pun inteded) change to the Translation Status Overview script, and we can add another keyword instead, maybe something like `English-only`? I don't know, you all can probably come up with a better word!

Reply:
Yes, we can _remove_ `minor` and add `en-only` if we like that!